Pods/Manifest.lock

```diff
@@ -0,0 +1,25 @@
+PODS:
```

Comment:
Exclude Pods/ directory from git by adding it to .gitignore and remove it from git